### PR TITLE
Conclude `Jogja Fun Cup`

### PR DESCRIPTION
**Pull request by Nivalyx**
apologies that this one took a while, i was only given the final list of all the replayers yesterday

image

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/JFC/1/en.md [hunk 1]
```diff
@@ -231,6 +241,20 @@ The Jogja Fun Cup is run by various osu! community members from Indonesia and be
...
 
+### Finals (week 2)
+
+Saturday, 10 August 2024:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | **Bakpia Kukus** | **7** | 1 | Ban HR2 | [#1](https://osu.ppy.sh/community/matches/115022242) |
+
+Sunday, 11 August 2024, Grand Final:
```
Comment (Walavouchey) [inline wiki/Tournaments/JFC/1/en.md:252]: this isn't marked elsewhere (it's the last match anyway)

```suggestion
Sunday, 11 August 2024:
```

Comment (Nivalyx) [reply wiki/Tournaments/JFC/1/en.md:252]: e i've been doing this everywhere else for years tho D:

image

image

Comment (Nivalyx) [reply wiki/Tournaments/JFC/1/en.md:252]: i think TicClick suggested this to me once and i've been adding this distinction ever since

Comment (Nivalyx) [reply wiki/Tournaments/JFC/1/en.md:252]: ah, alright then

**Comment by Nivalyx** [comment]
@Walavouchey can you recheck/merge this one if you have the time please, everything should be good to go 👀

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**